unified-runtime/scripts/core/device.yml

```diff
@@ -449,6 +449,8 @@ etors:
       desc: "[$x_bool_t] support the $xProgramSetSpecializationConstants entry point"
     - name: USE_NATIVE_ASSERT
       desc: "[$x_bool_t] return true if the device has a native assert implementation."
+    - name: BFLOAT16_CONVERSIONS_INTEL
```

Comment:
We should remove the `_INTEL` suffix here since we don't need to differentiate based on vendor for UR queries.

Reply:
Hi, @kbenzie 
I changed the enum name to "UR_BFLOAT_CONVERSIONS_NATIVE".
Thanks very much.